packages/core/src/components/SupportButton/SupportButton.tsx

```diff
@@ -94,6 +102,39 @@ export const SupportButton = ({ children }: PropsWithChildren<Props>) => {
     setPopoverOpen(false);
   };
 
+  const displayItems = () => {
```

Comment:
Is this switch really needed? I'm wondering if just combining the items into an array or the proper order first would make this a bit more succinct. That way the rendering logic for components is exactly the same.